src/quo2/components/notifications/notification_dot.cljs

```diff
@@ -4,12 +4,18 @@
 
```

Comment:
this component doesn't exist in the codebase but is used in many components for quo2. 
Perhaps we can move it to something like
src/quo2/components/common/notification_dot.cljs ? so we can mostly keep the mapping for design system components structure consistent with Figma?

Additionally can we put it into view and style files?